CircleSpawn/CircleSpawn/CircleSpawnController.swift

```diff
@@ -1,11 +1,90 @@
 import UIKit
 
-class CircleSpawnController: UIViewController {
+class CircleSpawnController: UIViewController, UIGestureRecognizerDelegate {
 
-	// TODO: Assignment 1
-
-	override func loadView() {
-		view = UIView()
-		view.backgroundColor = .white
-	}
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = .systemBackground
+    
+        let doubleTap = UITapGestureRecognizer(target: self, action: #selector(handleDoubleTap(sender:)))
+        doubleTap.delegate = self
+        doubleTap.numberOfTapsRequired = 2
+        view.addGestureRecognizer(doubleTap)
+        
+        let tripleTap = UITapGestureRecognizer(target: self, action: #selector(handleTripleTap(sender:)))
+        tripleTap.delegate = self
+        tripleTap.numberOfTapsRequired = 3
+        view.addGestureRecognizer(tripleTap)
+        
```

Comment:
Dlaczego tworzycie jeden triple tap na całą aplikację?

Pamiętajcie, że w przypadku doubletap, tapowanym widokiem jest główny widok view controllera (`self.view`), natomiast w przypadku triple tap tapowanym widokiem jest `circle`, więc to do niego powinien być dodawany ten gest (nowy triple tap do każdego circle).